vendors/sensecap/drivers/s2120/index.js

```diff
@@ -8,7 +8,7 @@ function decodeUplink(input) {
     }
 
     var bytes = input['bytes']
-    bytes = Buffer.from(bytes, 'utf8').toString()
+    bytes = Buffer.from(bytes, 'utf8').toString('hex')
```

Comment:
```suggestion
    bytes = Buffer.from(bytes).toString('hex')
```

the input.bytes is an array of integers, thus, this conversion does not work. You should not specify the source type as 'utf8'.